src/corva/models/merge/merge.py

```diff
@@ -25,23 +27,23 @@ class PartialRerunMergeEvent(CorvaBaseEvent):
         run_until: run until
     """
 
-    event_type: EventType
-    partial_well_rerun_id: int
-    rerun_mode: RerunMode
-    start: int
-    end: int
+    event_type: Optional[EventType]
+    partial_well_rerun_id: Optional[int]
+    rerun_mode: Optional[RerunMode]
+    start: Optional[int]
+    end: Optional[int]
     asset_id: int
     rerun_asset_id: int
     app_stream_id: int
     rerun_app_stream_id: int
-    version: int
-    app_id: int
-    app_key: str
+    version: Optional[int]
```

Comment:
I think this field can not be optional as well. It's part of the mechanism of backwards-compatibility when the shape of data is changed.